src/main/scala/no/ndla/audioapi/ComponentRegistry.scala

```diff
@@ -34,9 +49,21 @@ object ComponentRegistry
 
   ConnectionPool.singleton(new DataSourceConnectionPool(dataSource))
 
+  val amazonClient = new AmazonS3Client(new BasicAWSCredentials(AudioApiProperties.StorageAccessKey, AudioApiProperties.StorageSecretKey))
```

Comment:
Jeg tror nok at vi ønsker å endre til å ha credentials i properties til å bruke instans-profiler som har policies som gir nødvendig tilgang. Se f.eks code-deploy oppsettet. Det benytter policies for S3-buckets.